front/src/applications/stdcm/formatStcmConf.ts

```diff
@@ -2,12 +2,26 @@ import { Dispatch } from 'redux';
 import { TFunction } from 'i18next';
 
 import { STDCM_MODES, OsrdConfState } from 'applications/osrd/consts';
+import { TYPES_UNITS } from 'applications/osrd/components/Simulation/Allowances/consts';
 import { time2sec } from 'utils/timeManipulation';
 import { makeEnumBooleans } from 'utils/constants';
 
 import { ActionFailure } from 'reducers/main';
 import { ThunkAction } from 'types';
 
+type standardStdcmAllowanceForApi = {
```

Comment:
To be replaced when the openapi types are updated 
![Screenshot_20230113_114148](https://user-images.githubusercontent.com/24652545/212300817-c457a87a-a38c-4fb4-86e6-336624cde7b1.png)

Reply:
Agree